### ISSUE TITLE
Add bicycle_rental=docking_station to bike shares

### ISSUE THREAD

**Issue by andrewpmk** [osmlab/name-suggestion-index#9070]
Should we add the following tag: bicycle_rental=docking_station to bike sharing systems that use docking stations (e.g. Bike Share Toronto)?

This tool complains about this issue: https://lucaslongour.org/posts/2023-12-16-street-complete-progress-report/ Looks like StreetComplete is suggesting we fill in the bicycle_rental=* tag for bike sharing stations). Might prevent some StreetComplete users from making useless contributions.

**Comment by andrewpmk** [comment]
See https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbicycle_rental

bicycle_rental=docking_station is most common type but there are several others listed

Here's an example of a StreetComplete edit someone did to add bicycle_rental=docking_station https://www.openstreetmap.org/changeset/145554434#map=19/43.65189/-79.36522

**Comment by AntiCompositeNumber** [comment]
Makes sense. `bicylce_rental=` values are usually consistent within a system, and it's useful information.

**Comment by AntiCompositeNumber** [comment]
I ran an Overpass query for `amenity=bicycle_rental` with `brand:wikidata` and `bicycle_rental=*`, the results are at
https://gist.github.com/AntiCompositeNumber/930d463811b3198344578a373f11ea70.

**Comment by Cj-Malone** [comment]
For ones where it is unanimous, I support adding it. It's basically part of the category, like cuisine.

**Comment by andrewpmk** [comment]
I added it for the most common ones with over 100 entries (if not present):
3731 Q20687952	docking_station
   1380 Q1120762	docking_station
    799 Q807961	docking_station
    762 Q1034635	docking_station
    603 Q2974438	docking_station
    466 Q1833385	docking_station
    405 Q17402113	docking_station
    299 Q2413118	docking_station
    271 Q9092320	docking_station
    159 Q16971391	docking_station
    155 Q3564072	docking_station
    154 Q117066760	docking_station
    128 Q3142157	docking_station
    122 Q953933	docking_station
    107 Q4919302	docking_station
